ludwig/combiners/combiners.py

```diff
@@ -106,11 +149,24 @@ def __init__(
         if input_features and len(input_features) == 1 and fc_layers is None:
             self.supports_masking = True
 
+    @property
+    def _input_last_dimension_size(self) -> int:
```

Comment:
maybe a better name could be `concatenated_size` ?

Reply:
consolidated with `effective_input_shape`  to be part of `concatenated_shape` property